tests/integration/integration-test.py

```diff
@@ -44,9 +44,13 @@ def test_get_notes_path(self, endpoint='/notes'):
             response_data = response.json()['data']
             actual_note_id = response_data['id']
             self.assertEqual(actual_note_id, note_id)
+            # Validating the note received belong to the student id
+            student_id = (note_id.split('-'))
```

Comment:
I don't think the outer parenthesis is needed. Also, `note_id.split('-')` will actually give you a list of a string not only the `student_id`. I know you can get the student_id by accessing `student_id[0]`, but the naming here is not accurate. Another suggestion, I'd recommend using a regex to fetch the 9-digit OSU id instead of only depending on the `split` method so that you can make sure catch the error if note_id is invalid.